### PR TITLE
Typo on CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by anjaniacatus**
Hello, I'm landing here through the hacktoberfest. There was a time, that i have used a lot rsync with backuppc on Debian OS , that's why backintime catch my eyes. And i find it cool. Unfortunately, currently  i'm using  Mac OS. I hope next time i can contribute more on this project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ request is accepted.
   [Build and install via `make` system](#build-and-install-via-make-system-recommended)
   for further details.
 - Try to create new unit tests if appropriate. Use the style of regular Python
-  `unittest` rather than `pytest`. If you know the difference please try follow
+  `unittest` rather than `pytest`. If you don't know the difference, please try to follow
```
Comment (buhtz) [inline CONTRIBUTING.md]:
```suggestion
  `unittest` rather than `pytest`. If you know the difference, please try to follow
```

**Review by buhtz — CHANGES_REQUESTED** [review]
Hello Afenomamy,
On behalf of the team, thank you for your contribution and taking
the time to improve Back In Time. We appreciate it.

I was not aware that the project is somehow registered at Hacktoberfest. Can you show me a link or something?

Your PR looks fine beside the minor suggestion about the inverted meaning of the sentence. 

To find out how we usually tread PRs please read
["What happens after you opened a PullRequest (PR)?"](https://github.com/bit-team/backintime/blob/dev/CONTRIBUTING.md#what-happens-after-you-opened-a-pull-request-PR).

Best,
Christian

**Comment by anjaniacatus** [comment]
hum ok, it was a misunderstanding  from me then. I found Back In Time on this link : https://up-for-grabs.net/#/ . This link  is referenced in the https://hacktoberfest.com/participation/ under the "Start Contributing " title

**Comment by buhtz** [comment]
Ah, thank you. I wasn't aware that up-for-grabs is involved with Hacktober.

**Comment by daviewales** [comment]
I think that technically this would not count towards hacktoberfest unless the repository or PR have the appropriate opt-in topic or label:

From the hacktoberfest homepage:

> [participating] Your PR/MRs must be in a repo tagged with the “hacktoberfest” topic, or have the “hacktoberfest-accepted” label.

Up to the BiT team whether they want to do either of those things.

**Comment by anjaniacatus** [comment]
> I think that technically this would not count towards hacktoberfest unless the repository or PR have the appropriate opt-in topic or label: From the hacktoberfest homepage:
> [participating] Your PR/MRs must be in a repo tagged with the “hacktoberfest” topic, or have the “hacktoberfest-accepted” label.
> Up to the BiT team whether they want to do either of those things.

True, that's why is said it's a misunderstanding from me

**Review by buhtz — APPROVED** [review]

**Pull request merged**